pages/linux/sherlock.md

```diff
@@ -0,0 +1,32 @@
+# sherlock
+
+> Find usernames across social networks.
+
+- Display help:
+
+`sherlock --help`
+
+- Save result to file (if using single username):
```

Comment:
the result of what?

Reply:
Stores the links of the profiles with the provided username/usernames.

Reply:
Wait, `sherlock` opens the an UI app? a GTK window, I mean.

Reply:
Sherlock is a cli tool. With the --browse option, when it is finished with the searching, it opens the links to the profiles it has found in the default browser.

Reply:
Ok, perhaps is better:
```suggestion
- Search for a specific username on social networks saving the results to a file:
```